src/styles/01-atoms/_images/image/_image.scss

```diff
@@ -1,18 +1,30 @@
 @mixin image {
-  display: block;
-  height: auto;
-  max-width: 100%;
+    display: block;
+    height: auto;
+    max-width: 100%;
 }
 
 img,
 picture {
-  @include image;
+    @include image;
 }
 
 figure {
-  margin: 0; //override normalize
+    margin: 0; //override normalize
 }
 
 .figure__caption {
```

Comment:
It seems like this `.figure__caption` was unused.

Reply:
A lot of the CSS was pulled over from another project that was started before this one that uses a different tech stack and has different components. Once we're done with our Story clean-up milestone we should talk about removing unused CSS (I think one thing that will make that task a lot easier is getting this .scss files with their relevant component, which we want to tackle first).